lib/web_ui/dev/generate_builder_json.dart

```diff
@@ -113,7 +113,7 @@ Iterable<dynamic> _getAllTestSteps(List<TestSuite> suites) {
       suite.runConfig.browser == BrowserName.chrome ||
       suite.runConfig.browser == BrowserName.firefox
     ),
-    ..._getTestStepsForPlatform(suites, 'Mac', (TestSuite suite) =>
+    ..._getTestStepsForPlatform(suites, 'Mac', specificOS: 'Mac-12', (TestSuite suite) =>
```

Comment:
Can you add a TODO here that points to the issues we are tracking for the problems with the unit tests on Safari and the test harness?

Reply:
Should I assign any specific person in the TODO?

Reply:
Me. `TODO(jacksongardner):`